jstz_cli/src/octez.rs

```diff
@@ -36,6 +36,8 @@ impl OctezClient {
             &format!("http://127.0.0.1:{}", cfg.octez_node_rpc_port),
         ]);
 
+        command.env("TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER", "Y");
```

Comment:
Can you please write to the PR description or a comment what exactly does this do?

Reply:
`TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER` disables the a disclaimer that is logged on each octez-client command when using a 'unsafe network' such as a local sandbox (or a testnet)